testing/tests/DevExpress.ui.widgets.editors/datebox.mask.tests.js

```diff
@@ -1131,6 +1131,40 @@ module('Advanced caret', setupModule, () => {
         assert.deepEqual(this.keyboard.caret(), { start: 3, end: 5 }, 'caret was moved');
     });
 
+    test('Move caret to the next group if advanceCaret is changed at runtime', function(assert) {
```

Comment:
The `advanceCaret` option is:
- not documented
- always `true`
- never used

Let's just remove it.

Reply:
I going to remove it by separate PR

Reply:
Ok. But do we need adding a test for this option that will be removed in the next PR?